### PR TITLE
Updated lora inference instructions

### LINKED ISSUE #6906: LoRA dreambooth example is broken
### Describe the bug

Following the instructions from https://github.com/huggingface/diffusers/tree/main/examples/dreambooth do not yield personalised image. 
image

I noticed the problem in my own script, but decided to try and follow the instruction before raising it and it didn't work also. 
Looks like there was a recent commit to this folder

### Reproduction

All the code can be found in the notebook https://colab.research.google.com/drive/1rloHRIqgWapF2dXn58ygjgduKzBZbOtW?usp=sharing

### Logs

_No response_

### System Info

diffusers @  https://github.com/huggingface/diffusers@4a3d528
transformers==4.35.2
Python 3.10.12

### Who can help?

@yiyixuxu @DN6 @sayakpaul @patrickvonplaten

### PR DESCRIPTION
**Pull request by AlexUmnov**
# What does this PR do?

Fixes #6906

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [ ] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [ ] Was this discussed/approved via a GitHub issue or the [forum](https://discuss.huggingface.co/c/discussion-related-to-httpsgithubcomhuggingfacediffusers/63)? Please add a link to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/diffusers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@sayakpaul

### CODE REVIEW

File: examples/dreambooth/README.md
Comment (sayakpaul) [inline examples/dreambooth/README.md:381]: Let's not change these things please.

File: examples/dreambooth/README.md
Comment (sayakpaul) [inline examples/dreambooth/README.md:387]: Let's simplify it:

```suggestion
pipe.load_lora_weights("path-to-the-lora-checkpoint")
```

**Review by sayakpaul — COMMENTED** [review]
Thanks! Left some comments.

File: examples/dreambooth/README.md
Comment (AlexUmnov) [reply examples/dreambooth/README.md:381]: I can remove `os`, but I think it's better to switch it to StableDiffusionPipeline
It's a sd pipeline after all. And you don't need this wizardry with DPMSolverMultistepScheduler

**Comment by AlexUmnov** [comment]
@sayakpaul  let me know if you want to keep it a Diffusion pipeline instead of StableDiffusion

File: examples/dreambooth/README.md
Comment (sayakpaul) [reply examples/dreambooth/README.md]: It's okay to use `DiffusionPipeline` here since it's more generic. I am not sure how that is related to any wizardry.

File: examples/dreambooth/README.md
Comment (sayakpaul) [inline examples/dreambooth/README.md:380]:
```suggestion
pipe = DiffusionPipeline.from_pretrained("base-model-name").to("cuda")
```

**Review by sayakpaul — APPROVED** [review]
Single suggestion. After that we can merge.

**Comment by sayakpaul** [comment]
Will merge once the CI is green.

**Pull request merged**
